### ISSUE TITLE
The question on PointNetFeaturePropagation class

### ISSUE THREAD

**Issue by HustQBW** [qq456cvb/Point-Transformers#30]
https://github.com/qq456cvb/Point-Transformers/blob/5caf2caede06b452592a16e2b342df48d24fbdd3/pointnet_util.py#L261

In this class, there is one line code as following:

`interpolated_points = torch.sum(index_points(points2, idx) * weight.view(B, N, 3, 1), dim=2)`

I think points2 should be replaced by points1. 

The variable "idx" refers to the top3 index of points1 whose distance is short to points2, and the “index_points” function is to find k(k=3) nearest neighbor of each point in points2 , which is found in  points1.

Are there any questions in my understanding? I really sincerely want to get your answer, thanks !
